cloud/amazon/ec2_vol.py

```diff
@@ -332,7 +338,10 @@ def create_volume(module, ec2, zone):
     if volume is None:
         try:
             if boto_supports_volume_encryption():
-                volume = ec2.create_volume(volume_size, zone, snapshot, volume_type, iops, encrypted)
+                if kms_key_id:
```

Comment:
For most things I'd prob be ok with what you're saying but the last thing I would want is for a user to think that a kms key was being used for encryption, as a result of a non-error during executing, when in fact it isnt.